src/script/main/app.ts

```diff
@@ -398,6 +400,15 @@ export class App {
         await e2eiHandler.attemptEnrollment();
       }
 
+      teamRepository.on('featureConfigUpdated', async configUpdate => {
+        const {type} = getTeamFeatureUpdate(configUpdate, FEATURE_KEY.MLSE2EID);
+
+        if (type !== FeatureUpdateType.UNCHANGED) {
+          const client = await configureE2EI(this.logger, configUpdate.newFeatureList);
+          return client?.attemptEnrollment();
+        }
+      });
```

Comment:
Humm this would probably be better somewhere in a hook that will have an unmount cleanup. 

Also it doesn't really make sense to subscribe before the app is fully loaded. 

So I'd suggest isolating this to a hook that is initialize by the `AppMain` component. WDYT?

Reply:
Great idea 👌

Reply:
Addressed with https://github.com/wireapp/wire-webapp/pull/16675/commits/637e9fdee3f27b64f1e7fdd042e12f5863cf73ca